src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -2353,11 +2353,11 @@ namespace InternalHeatGains {
                     if (AnyEnergyManagementSystemInModel) {
                         SetupEMSActuator("GasEquipment",
                                          ZoneGas(Loop).Name,
-                                         "Gas Power Level",
+                                         "NaturalGas Rate",
                                          "[W]",
                                          ZoneGas(Loop).EMSZoneEquipOverrideOn,
                                          ZoneGas(Loop).EMSEquipPower);
-                        SetupEMSInternalVariable("Gas Process Power Design Level", ZoneGas(Loop).Name, "[W]", ZoneGas(Loop).DesignLevel);
+                        SetupEMSInternalVariable("NaturalGas Process Power Design Level", ZoneGas(Loop).Name, "[W]", ZoneGas(Loop).DesignLevel);
```

Comment:
This is linked to the doc [comment above](https://github.com/NREL/EnergyPlus/pull/8304/files#r496194671) does it stay or does it go. I'm good with let it go.

Reply:
Took it back out.